snapshots/war/2024-01-23/prohibition.csv.dvc

```diff
@@ -0,0 +1,32 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/origin/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Treaties Database
+    description: |-
+      Since 1978, the United Nations has been publishing the status of multilateral Arms Regulation and Disarmament Agreements, presenting data on signatories, parties of relevant agreements and the texts of the agreements themselves.
+    title_snapshot: Treaties Database - Treaty on the Prohibition of Nuclear Weapons
+    description_snapshot: |-
+      This dataset contains the list of countries that have participated in the Treaty on the Prohibition of Nuclear Weapons.
+    date_published: 2024-01-23
```

Comment:
I haven't found anything like a latest date of update. But they do update regularly. I thought I could use the date of the latest insertion in the table, but for some older treaties, that may be years ago (which would be misleading, since the table is still up-to-date today).
I suppose UNODA is not going to ignore if a country signs or changes its status on any of these treaties, so the table is probably always up to date.